src/nameclaim.cpp

```diff
@@ -173,3 +175,22 @@ size_t ClaimNameSize(const CScript& scriptIn)
         return vvchParams[0].size();
     }
 }
+
+CAmount CalcMinClaimTrieFee(const CTransaction& tx, const CAmount &minFeePerNameClaimChar)
+{
+    CAmount min_fee = 0;
+    BOOST_FOREACH(const CTxOut& txout, tx.vout)
```

Comment:
Why not skip the loop when minFeePerNameClaimChar is 0?

Reply:
good idea